ember-amount-input/package.json

```diff
@@ -22,9 +22,12 @@
     "lint:fix": "concurrently 'pnpm:lint:*:fix' --names 'fix:'",
     "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
     "lint:js": "eslint . --cache",
+    "lint:types": "glint",
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",
-    "start": "rollup --config --watch",
+    "start": "concurrently 'pnpm:start:*'",
+    "start:js": "rollup --config --watch --no-watch.clearScreen",
```

Comment:
Why are we specifying the `clearScreen` option with `--no-watch.clearScreen` instead of `--watch.clearScreen`? https://rollupjs.org/configuration-options/#watch-clearscreen

Maybe because `watch.clearScreen` will always clear the screen while `no-watch.clearScreen` will just do it at the beginning of the build? This is not clear from the doc, I'm sure you have more context 😢

Reply:
We need to specify this option, otherwise, rollup and glint will fight to clear the screen. And in watch mode, glint always clears the screen. So to avoid conflict, we have to ask rollup to not clear the screen on each rebuild

Reply:
Clear, thanks @vscav 🙏